packages/eslint-config-react-native-community/index.js

```diff
@@ -12,6 +12,8 @@ module.exports = {
 
   env: {
     es6: true,
+    jest: true,
```

Comment:
Shouldn't it be under `overrides` like:

```js
  overrides: {
    files: ['**/__tests__/**/*.js', '**/?(*.)(spec|test).js'],
    env: {
      jest: true,
      'jest/globals': true,
    },
  },
```

I think adding all jest globals to the normal env isn't great.